libafl_libfuzzer/libafl_libfuzzer_runtime/src/merge.rs

```diff
@@ -198,9 +198,15 @@ pub fn merge(
 
     // In case the corpus is empty (on first run) or crashed while loading, reset
     if state.must_load_initial_inputs() && !options.dirs().is_empty() {
+        let loaded_dirs = options
```

Comment:
Maybe do this once and store it to state, because you still want the same order after restart(?)

Reply:
Order will always be consistent here, since it refers to the command line arg order.